test/unit/lib/plugins/aws/package/compile/events/alb/lib/validate.test.js

```diff
@@ -144,6 +144,78 @@ describe('#validate()', () => {
       },
     });
   });
+  it('should accept alb header as both array and key-value object', () => {
```

Comment:
If those tests confirm on same things as those configured with `runServerless` then let's just remove them, otherwise let's replace them with `runServerless` based variants  (`alb/index.test.js` file)